packages/main/src/themes/BusyIndicator.css

```diff
@@ -128,10 +128,17 @@
 
 .ui5-busy-indicator-text {
 	width: 100%;
-	margin-top: .25rem;
 	text-align: center;
 }
 
+.ui5-busy-indicator-text-placement-top {
```

Comment:
Some points for improvement.

Instead of creating, switching classes:

```js
get classes() {
   textPosition: {
		"ui5-busy-indicator-text-placement-top ": this.textPlacement === BusyIndicatorTextPlacement.Top,
		"ui5-busy-indicator-text-placement-bottom ": this.textPlacement === BusyIndicatorTextPlacement.Bottom,
    },
}
```
```css
.ui5-busy-indicator-text-placement-top {
   margin-bottom: 0.5rem
}
.ui5-busy-indicator-text-placement-bottom {
   margin-top: 0.5rem;
}
```

you can just add styles based on the attribute:
```css 
:host([text-placement="Top"]) .ui5-busy-indicator-text {
   margin-bottom: 0.5rem
}

:host(:not([text-placement)) .ui5-busy-indicator-text,
:host([text-placement="Bottom"]) .ui5-busy-indicator-text {
     margin-top: 0.5rem;
}
```
We do it like when we can, when the conditions are simple enough to cover with attribute selectors.

Reply:
#8482 taking care of improvment